repositories.bzl

```diff
@@ -0,0 +1,32 @@
+#
+#  Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Contains the external dependencies
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+def fuzzing_rule_dependencies():
```

Comment:
Isn't rules_cc also used? (How does it get added without the workspace?)

Reply:
From the `rules_cc` [readme](https://github.com/bazelbuild/rules_cc#getting-started), I should add it, but the interesting thing is that the project runs without loading `rules_cc` so long. Let me add that part!

Reply:
> There is no need to use rules from this repository just yet. If you want to use rules_cc anyway, add the following to your WORKSPACE file
http_archive(
    name = "rules_cc",
    urls = ["https://github.com/bazelbuild/rules_cc/archive/TODO"],
    sha256 = "TODO",
)

The url listed above is a `404` link. I think the `rules_cc` is shipped together with the bazel, thus no need to explicitly `http_archive` it?

Reply:
Added the `cc_rules` codes given in the repository, got `404` error returned by `http_archive`. Seems we can use the `rules_cc` directly for now.